cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -807,3 +807,7 @@ def test_library_content_on_course_export_import(self, publish_item):
             self.source_course.location,
             dest_course.location
         )
+
+    def test_branch_settings(self):
```

Comment:
Can you move this test to the `modulestore` suite of tests in `common`, please, and test it with both old-mongo and split-mongo?